content/modules/enterprise-capabilities.md

```diff
@@ -17,7 +17,6 @@ The following table shows which modules are supported by Redis Enterprise Softwa
 
 | Module | Redis Enterprise<br/>Software | Redis Enterprise<br/>Cloud |
 |:-------|:-------------------------:|:-----------------------:|
-| [Redis Stack]({{<relref "/modules/redis-stack">}}) | &#x2705; Yes[^4] | &#x2705; Yes[^4] |
```

Comment:
If we delete 4, should the new not be 4 rather than 5 @rrelledge (not an expert)

Reply:
Updated.